services/fdb/server/fdbcli.go

```diff
@@ -1155,6 +1157,35 @@ func parseFDBCLISuspend(req *pb.FDBCLISuspend) ([]string, []captureLogs, error)
 	return args, nil, nil
 }
 
+func parseFDBCLITenantEmergencyMove(req *pb.FDBCLITenantEmergencyMove) ([]string, []captureLogs, error) {
+	args := []string{"tenant emergency_move"}
```

Comment:
The actual argument to fdbcli needs to have a space because `emergency_move` is a subcommand of the `tenant` umbrella of operations. Will the second option pass both strings with a space in between?

Reply:
Normally when you type in a command line, the shell takes care of splitting the command on whitespace.

That doesn't happen with golang's os/exec package. Instead, what you have here is the equivalent of passing in `/bin/fdbcli "tenant emergency_move"` on the shell.